monitor.py

```diff
@@ -29,14 +29,28 @@ def fetch_release_date(release):
                 sys.exit(1)
         if response.status_code == 200:
             soup = BeautifulSoup(response.text, 'html.parser')
+            form = soup.find("form", {"method": "GET"})
```

Comment:
Verify if we can have only one form in a page and if not handle that multiple form elements as well